### PR TITLE
chore: Matic configuration + TPRV2

### PR DESCRIPTION
**Pull request by fzavalia**

### CODE REVIEW

File: subgraph.template.yaml [hunk 1]
```diff
@@ -37,10 +37,6 @@ dataSources:
           handler: handleThirdPartyItemSlotsBought
         - event: ThirdPartyReviewedWithRoot(string,bytes32,bool,address)
           handler: handleThirdPartyReviewedWithRoot
-        - event: ItemAdded(string,string,string,bool,address)
-          handler: handleItemAdded
-        - event: ItemUpdated(string,string,string,address)
-          handler: handleItemUpdated
```
Comment (LautaroPetaccio) [inline subgraph.template.yaml:43]: Don't we need to remove `ItemReviewed` as well?
Should we delete the item entity too?
```
      entities:
        - ThirdParty
        - Item
```

File: src/handlers/thirdparty.ts [hunk 2]
```diff
@@ -150,76 +146,6 @@ export function handleThirdPartyReviewedWithRoot(
...
-
-  item = setItemSearchFields(item)
-
-  thirdParty.save()
-  item.save()
-
-  let metric = buildCountFromItem()
-  metric.save()
-}
-
```
Comment (LautaroPetaccio) [inline src/handlers/thirdparty.ts:222]: Should we remove `handleItemReviewed` as well?

**Review by LautaroPetaccio — CHANGES_REQUESTED** [review]
There's some code related to the items and to their metadata that won't be used anymore, I think that we should remove that as well.

**Comment by fzavalia** [comment]
@LautaroPetaccio Removed both the handleItemReviewed and everything related to the Item entity as they seem unnecessary. Remove only ItemAdded and ItemUpdated to just mirror what was removed from the contract.

The contract is still emitting https://github.com/decentraland/wearables-contracts/blob/feat/new-registry-version/contracts/registries/ThirdPartyRegistryV2.sol#L418 the reviewed event, however that bunch of code is now reachable by not being allowed to add items. Yet we didn't remove it to avoid messing up the audit.

We don't have any limitations with the graph so I believe its a good idea to remove unused things from here.

Thanks!

**Review by LautaroPetaccio — APPROVED** [review]

**Pull request merged**